arches/app/templates/views/graph/model-history.htm

```diff
@@ -0,0 +1,195 @@
+                                <span data-bind="text: $parent['publishedUserData']()[$data.user_id]['last_name']">,</span>
+                            <!-- /ko -->
+                            <!-- ko if: $parent['publishedUserData']()[$data.user_id]['first_name'] || $parent['publishedUserData']()[$data.user_id]['last_name'] -->
+                                <span>|</span>
+                            <!-- /ko -->
+                            <span data-bind="text: $parent['publishedUserData']()[$data.user_id]['username']"></span>
+                        <!-- /ko -->
+                    </div>
+                </div>
+
+                <div style="justify-content: center; display: flex; padding-right: 40px;">
+                    <div>
+                        <span><strong>{% trans "Number of Resources:" %}</strong></span>
+                        <span data-bind="text: $parent['graphPublicationResourceInstanceCount']()[$data['publicationid']]"></span>
+                    </div>
+                </div>
+
+                <!-- ko ifnot: $data['publicationid'] === $parent.graphPublicationId() -->
+                    <div style="display: flex; align-items: center; width: 100%; overflow-x: hidden; padding-right: 40px;">
+                        <div style='width: 60px; flex: none;'>
+                            <strong>{% trans "Notes:" %}</strong>
+                        </div>
+    
+                        <div 
+                            style="
+                                overflow-y: hidden; 
+                            "
+                        >
+                            <div 
+                                style="    
+                                    overflow: hidden;
+                                    white-space: nowrap;
+                                    text-overflow: ellipsis;
+                                " 
+                                data-bind="text: $data['notes'] || {% quoted_trans 'None' %}" 
+                                readonly
+                            ></div>
+                        </div>
+                    </div>
+                <!-- /ko -->
+            </div>
+
+            <!-- ko if: $data['publicationid'] === $parent.graphPublicationId() -->
+                <div 
+                    style="
+                        display: flex; 
+                        background-color: #fff; 
+                        padding: 0px 60px 20px 154px; 
+                        border-left: 1px solid #ddd; 
+                        border-right: 1px solid #ddd; 
+                        cursor: pointer;
+                    "
+                >
+                    <div style="width: 60px; flex: none;">
+                        <strong>{% trans "Notes:" %}</strong>
+                    </div>
+                    <div 
+                        style="
+                            overflow-y: scroll;
```

Comment:
```suggestion
                            overflow-y: auto;
```
Seems to prevent this situation on MacOS:
<img width="782" alt="always-scrollbar" src="https://github.com/archesproject/arches/assets/38668450/2a936b98-8488-4932-86bb-3cc51a5af01c">

Reply:
can't seem to repro but setting it to auto doesn't affect the UX negatively for me, so happy to include this